rpcs3/Emu/RSX/Common/VertexProgramDecompiler.cpp

```diff
@@ -283,7 +283,7 @@ std::string VertexProgramDecompiler::GetCond()
 std::string VertexProgramDecompiler::GetOptionalBranchCond()
 {
 	std::string cond_operator = d3.brb_cond_true ? " != " : " == ";
-	std::string cond = "(transform_branch_bits & (1 << " + std::to_string(d3.branch_index) + "))" + cond_operator + "0";
+	std::string cond = "(transform_branch_bits & (1u << " + std::to_string(d3.branch_index) + "))" + cond_operator + "0";
```

Comment:
does the 0 also need cast to uint? the expression expands to 
if ((transform_branch_bits & const) == 0) for example.

Reply:
Nvm, it shouldn't be an issue.

Reply:
I only got debug_output nag regarding bitwise operations

Reply:
Yea, I understand why, its type "promotion". The bitwise part was what was mismatched, comparing against const seems to be properly defined.